test/parse-result.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * Tests for parse result namespace elements, including all their
+ * convenience properties and methods.
+ */
+
+import chai, {Assertion, expect} from 'chai';
+
+import minim from 'minim';
+import minimApiDescription from 'minim-api-description';
+import minimParseResult from '../src/parse-result';
+
+const namespace = minim.namespace()
+  .use(minimApiDescription)
+  .use(minimParseResult);
+
+const Category = namespace.getElementClass('category');
+const ParseResult = namespace.getElementClass('parseResult');
+const Annotation = namespace.getElementClass('annotation');
+const SourceMap = namespace.getElementClass('sourceMap');
```

Comment:
This is a fantastic idea, but is probably going to need to use the element names (maybe Pascal cased like above). We just can't assume the names of registered classes, but we can assume the element name. I'll play around with this idea a bit!